app/views/flexible_criteria/index.html.erb

```diff
@@ -74,7 +74,7 @@
   <h2><%= t('flexible_criteria.download.title') %></h2>
   <p><%= t('flexible_criteria.download.prompt') %></p>
 
-  <%= button_to t('flexible_criteria.download.link'),
+  <%= link_to t('flexible_criteria.download.link'),
                 { controller:'flexible_criteria',
```

Comment:
This is a bit picky, but please adjust the indentation of these following lines.